docs/dev_guide/diagnostic_ref/dpct1099.rst

```diff
@@ -41,8 +41,66 @@ If you received a warning for ``dpct::fft::fft_engine::create()`` or
 
 * If you are using an internal workspace and do not care about performance, you
   can ignore this warning.
-* If you do care about performance, specify the ``direction_and_placement``
-  parameter correctly.
+* If you do care about performance or using external workspace, specify the
```

Comment:
Suggest "If you do care about performance, or if you are using an external workspace,"